cogs/polling.py

```diff
@@ -0,0 +1,124 @@
+import discord
+import asyncio
+from discord.ext import commands
+import random
+from discord.ext.commands.cooldowns import BucketType
+
+
+class Poll(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+        self.emojiLetters = [
+            "\N{REGIONAL INDICATOR SYMBOL LETTER A}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER B}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER C}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER D}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER E}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER F}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER G}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER H}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER I}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER J}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER K}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER L}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER M}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER N}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER O}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER P}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER Q}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER R}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER S}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER T}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER U}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER V}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER W}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER X}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER Y}",
+            "\N{REGIONAL INDICATOR SYMBOL LETTER Z}"
+        ]
+
+    # parses the title, which should be in between curly brackets ('{ title }')
+    def find_title(self, message):
+        # this is the index of the first character of the title
+        first = message.find('{') + 1
+        # index of the last character of the title
+        last = message.find('}')
+        if first == 0 or last == -1:
+            return "Not using the command correctly"
+        return message[first:last]
+
+    # parses the options (recursively), which should be in between square brackets ('[ option n ]')
+    def find_options(self, message, options):
+        # first index of the first character of the option
+        first = message.find('[') + 1
+        # index of the last character of the title
+        last = message.find(']')
+        if (first == 0 or last == -1):
+            if len(options) < 2:
+                return "Not using the command correctly"
+            else:
+                return options
```

Comment:
Pylint and codacy: cogs/polling.py:58:12: R1705: Unnecessary "else" after "return" (no-else-return)

```suggestion
            return options
```